### PR TITLE
InterouteVDC driver added.

### PR DESCRIPTION
**Pull request by radu-stefanache**
Adding the Interoute VDC machine-driver.

### CODE REVIEW

File: machine-templates/interoutevdc/0/uiUrl [hunk 1]
```diff
@@ -0,0 +1 @@
+https://raw.githubusercontent.com/Interoute/ui-driver-interoutevdc/v1.0/dist/component.js
```
Comment (vincent99) [inline machine-templates/interoutevdc/0/uiUrl]: This doesn't work... github returns `Content-Type: text/plain` and browsers will refuse to interpret as JS.  Also only the domain that the driver is loaded from is automatically whitelisted, so the user would need to add `raw.githubusercontent.com` to the `api.proxy.whitelist` to allow it to be loaded.

**Comment by vincent99** [comment]
@radu-stefanache 

image

**Comment by radu-stefanache** [comment]
@vincent99  It works for me, I've tried it from multiple public facing servers (various zones/countries). Could be your browser/cors settings (if using)/vpn ?

**Comment by vincent99** [comment]
That's a HTML response from the server (IIS), browser and CORS not relevant.  I didn't change anything but it's working now.

**Pull request merged**